global.json

```diff
@@ -1,6 +1,6 @@
 {
     "projects": [ "src" ],
     "sdk": {
-        "version": "1.0.0-preview2-003131"
+        "version": "1.0.3"
```

Comment:
Time to updated to a release version of .NET Core. Developers may need to download the updated package if you don't have it.